api/controllers/node.js

```diff
@@ -246,4 +262,54 @@ NodeController.getPooledTransactions = function(context, next) {
 	);
 };
 
+/**
+ * Get the forging status of a delegate.
+ *
+ * @param {string} publicKey - Public key of delegate
+ * @returns {Promise<object>}
+ * @private
+ */
+async function _getForgingStatus(publicKey) {
+	const keyPairs = library.modules.delegates.getForgersKeyPairs();
+	const internalForgers = library.config.forging.delegates;
```

Comment:
why the new name `internalForgers`? it can be `forgingDelegates`?

Reply:
Updated